src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -682,6 +682,35 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
             return true; // to skip internal onPaste handler
         }
 
+        // Safari `Insert from iPhone or iPad`
+        // data.getData("text/html") returns a string like: <img src="blob:https://...">
+        if (data.types.includes("text/html")) {
+            const img_element_str=data.getData("text/html");
+            const img_element = document.createElement("div");
+            img_element.innerHTML = img_element_str;
```

Comment:
This is vulnerable to XSS attacks, if another site copies `text/html` containing a script tag and tells you to paste it into your Element client to "Unlock Pro Mode" or something it could steal all your secrets. You should use https://developer.mozilla.org/en-US/docs/Web/API/DOMParser

Reply:
Nope, it'd deploy it for you too (once I approve it) - but right now Github is on fire https://www.githubstatus.com/ so isn't running our workflows

Reply:
Indeed on fire...

My yarn can't download the package.json from this branch 

😠

Reply:
Should be fixed in latest commit.